homeassistant/components/totalconnect/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/totalconnect",
   "iot_class": "cloud_polling",
   "loggers": ["total_connect_client"],
-  "requirements": ["total-connect-client==2023.2"]
+  "requirements": ["total-connect-client==2023.12.1"]
```

Comment:
Oh and please bump this in a separate PR

Reply:
Even when bumping is required for this PR to work?  If the "bump PR" gets approved later, the bypass functions added here will fail.

Reply:
This PR would fail without that bump, but they are usually approved waaaay quicker